tests/template_tests/test_engine.py

```diff
@@ -21,6 +22,45 @@ def test_basic_context(self):
         )
 
 
+class GetDefaultTest(SimpleTestCase):
+
+    def test_single_dtl_engine_configured(self):
```

Comment:
I'd put this test after the next one, so you get the very logical order 0 engines, 1 engine, 2 engines. Also, you might want to explicitly set the `TEMPLATES` setting for this test, too
* as to not rely on the number of engines in the global setting (although that is unlikely to change)
* (more importantly) to communicate to the reader what scenario is being tested in comparison to the other two tests